bin/do_cf.py

```diff
@@ -101,6 +101,10 @@ def corr_func(p):
 
     cosmo = constants.cosmo(args.fid_Om)
 
+    lambda_abs  = constants.absorber_IGM[args.lambda_abs]
+    if (args.lambda_abs2) : lambda_abs2 = constants.absorber_IGM[args.lambda_abs2]
```

Comment:
`if not args.lambda_abs2 is None`

Reply:
I have seen that you put this in other places in the code, why ? 
This is shorter to write "if (args.lambda_abs2)", there is a difference with "if not args.lambda_abs2 is None" ?

Reply:
no difference, do as you want